validate.lic

```diff
@@ -88,6 +88,19 @@ class DRYamlValidator
     bad_names.each { |skill_name| error("Invalid weapon_training: skill name '#{skill_name}'") }
     echo("Valid skills are #{@valid_weapon_skills}")
   end
+  
+  def assert_that_combat_spell_training_spells_are_skills(settings)
+    return unless settings.combat_spell_training
+    
+    valid_combat_spell_training_skills = %w[Augmentation Warding Utility]
+
+    bad_names = settings.combat_spell_training.keys
+                        .reject { |skill_name| valid_combat_spell_training_skills.include?(skill_name) }
```

Comment:
I think this is a little more clear, what do you think? (It follows the convention of the rest of the software too, normally layered method calls are on a separate line like this.)
```
bad_names = settings.combat_spell_training
                                   .key
                                   .reject { |skill_name| valid_combat_spell_training_skills.include?(skill_name) }
```

Reply:
I am happy to follow the convention in the rest of code base, no problem. I like the in lining of the each too, I will make that change.